openstf_dbc/utils/utils.py

```diff
@@ -0,0 +1,13 @@
+# SPDX-FileCopyrightText: 2021 2017-2021 Alliander N.V. <korte.termijn.prognoses@alliander.com>
+#
+# SPDX-License-Identifier: MPL-2.0
+import pandas as pd
+
+
+def get_datetime_index(start, end, freq):
+    return pd.date_range(
+        start=start,
+        end=end,
+        freq=freq,
+        tz="UTC",
+    )
```

Comment:
My preference would be to not define such a custom wrapper around basic pandas functionality.
I get that you defined this function since you want to make sure a timezone is set. But still.

Reply:
How would you suggest to ensure/enforce that we use the same of datetime index everywhere? I've ran in to various issues in the past where index where different and caused issues, aware vs unaware but also different datatypes or different types of utc timezones etc.

Reply:
Checking it explicitly in unit tests in crucial parts of the code I guess.
I don't like the separate function since it makes it slightly less visible what is happening higher up in the code. Also, if someone is not using the correct timezone-awareness because they don't know that's important, they likely won't know about this function either. What I mean; that this function exists does not enforce the usage of it either.

Reply:
Okay, I think its better to leave it out for now in that case. I just thought something like this could improve the situation a bit. But you made some valid points as well and its not really part of this feature.